curry_bot.py

```diff
@@ -8,13 +8,23 @@
 
 import re
 import time
+import math
 
 
 TOKEN = get_token()
+RANDO_BASE = 'https://adrando.com/'
 RANDO_LINKS = ['https://adrando.com/']
 client = Bot(command_prefix='!', status=Status.online, activity=Game("Azure Dreams"))
 BOT_ID = '631144975366619146'
 COUNTDOWN_START = 10
+SEEDS_TO_GENERATE = 3
```

Comment:
Perhaps want to make this customisable? E.g. final matches will be best of 5. And sometimes a single seed is needed if runners are doing a restream at a different time.

Reply:
Implemented in 1ef78ba.